src/components/ui/PageContentContainer.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+import { respondFrom, breakpoints } from "../../styles";
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const Wrapper = styled.div`
+  padding: 0 16px;
+
+  ${respondFrom(breakpoints.desktop)`
+      max-width: 1232px;
```

Comment:
1232 looks like a magic  number. I would suggest creating a global const (e.g. PAGE_CONTENT_WIDTH = 1232) at the top of this file and then using it here (in a template string). it's quite probable that you will be using this value in other places

Reply:
I create new variable for content page with